workflow-controller/controller-service/src/main/java/Converters/ProfileConverter.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package Converters;
+
+import io.hyscale.commons.constants.ToolConstants;
+import io.hyscale.commons.constants.ValidationConstants;
+import io.hyscale.commons.models.Activity;
+import io.hyscale.controller.util.ServiceProfileUtil;
+import io.hyscale.servicespec.commons.activity.ServiceSpecActivity;
+import io.hyscale.servicespec.commons.fields.HyscaleSpecFields;
+import io.hyscale.servicespec.commons.model.service.Profile;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+
+/*
+Provides parameters and funtions such as profile reference schema,Regex for profile file naming,
+data validation and respective warning and error messages  for profile file validation.
+ */
+public class ProfileConverter extends Converter {
+    private static final String PROFILE_SCHEMA_FILE = "/hprof/" + ToolConstants.RELEASE + "/profile-spec.json";
```

Comment:
Should we look for a way to pick this value from config?

Reply:
yes the release number should be taken from a properties file generated by the build.